tests/e2e/file_volume_statefulsets.go

```diff
@@ -761,7 +761,7 @@ var _ = ginkgo.Describe("[csi-file-vanilla] File Volume statefulset", func() {
 			deleteService(namespace, client, service)
 		}()
 		ginkgo.By("Creating statefulset with replica 3 and a deployment")
-		statefulset, _, volumesBeforeScaleUp := createStsDeployment(ctx, client, namespace, sc, true,
+		statefulset, _, volumesBeforeScaleUp := createStsDeployment(ctx, client, namespace, sc, false,
```

Comment:
keep deployment as well

Reply:
Deployment is not needed for this test, i guess here we target file volume testing with statefulset.
Creation of deployment causing test failures, which this pr target to fix